Mlem/API/Requests/Post/CreatePost.swift

```diff
@@ -32,6 +33,7 @@ struct CreatePostRequest: APIPostRequest {
         name: String,
         nsfw: Bool?,
         body: String?,
+        // TODO change to `URL?`
```

Comment:
Hm? What do you mean?

This is more because im not entirely sure if they'd be valid `URL`s, but I added a todo here to check later.

Reply:
ah fair enough, I just noticed I've added a TODO further down in the same file because what we get passed isn't sanitised, so but to leave it as it'd require work at the UI layer to send a guaranteed one in 👍